catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/history/Historian.java

```diff
@@ -205,15 +219,22 @@ public DeleteResponse version(DeleteResponse deleteResponse)
         }
         setSkipFlag(deleteResponse);
 
-        Map<String, List<ContentItem>> contentItems = getContentItems(deleteResponse);
+        List<Metacard> deletedMetacards = deleteResponse.getDeletedMetacards()
+                .stream()
+                .filter(isVersionOrDeleted.negate())
+                .collect(Collectors.toList());
+
+        // [ContentItem.getId: content items]
+        Map<String, List<ContentItem>> contentItems = getContent(getReadStorageRequests(
```

Comment:
This is a different, overloaded method. the previous one was just to remove some boiler plate and then call this one.  Also this is used in 3 places, where that other one was just that once.